### PR TITLE
Cdpt 243 validation when creating ico overturned case

### PR DESCRIPTION
**Pull request by MeenaModhvadia**
## Description
Add descriptive validation message when creating an ico overturned case.
## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [ ] (3) Tests passing
* [*] (4) Branch ready to be merged (not work in progress)
* [*] (5) No superfluous changes in diff
* [*] (6) No TODO's without new ticket numbers
* [*] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [ ] (8) Data migration script is created if any of letter templates is changed

### Screenshots
Screenshot 2023-04-18 at 22 49 37

Screenshot 2023-04-18 at 22 34 36

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CDPT-417?atlOrigin=eyJpIjoiMDkxYzU3MThmNzZlNDZlYTllOTViMjlhM2VkMDE4YzAiLCJwIjoiaiJ9

### CODE REVIEW

File: spec/models/case/overturned_ico/foi_spec.rb [hunk 1]
```diff
@@ -108,14 +108,13 @@
           it 'errors' do
             new_case.received_date = 41.days.ago
             expect(new_case).not_to be_valid
-            expect(new_case.errors[:received_date]).to eq ['decision received from ICO must be within 40 days of today']
-          end
+            expect(new_case.errors[:received_date]).to eq ["of decision from ICO must be within 40 days of today"]
```
Comment (vertism) [inline spec/models/case/overturned_ico/foi_spec.rb:111]: FYI, `end` has been removed
```suggestion
            expect(new_case.errors[:received_date]).to eq ["of decision from ICO must be within 40 days of today"]
          end
```

**Review by njpreddy — APPROVED** [review]
LGTM :-)

**Review by njpreddy — APPROVED** [review]
LGTM :-)

**Review by vertism — APPROVED** [review]
👍

**Pull request merged**